recipes/_repositories.rb

```diff
@@ -9,6 +9,14 @@
         gpgkey node.attribute['confluent']['repository']['rpm']['key']
         action :create
     end
+    if version >= '3.0'
```

Comment:
If I understand the documentation right this is only the case for rpm/yum and not deb?